_components/validation/guidance/implementation.md

```diff
@@ -1,7 +1,7 @@
-- The validation component is intended primarily for usability, not as a robust security solution, since all the validation logic occurs on the client-side. The validation should be "mirrored" on the server-side for security purposes.
-- Input fields which have custom validation logic can automatically provide helpful feedback to users if they are assigned a `data-validation-element` attribute set to a CSS selector that uniquely identifies a `.usa-checklist`, e.g. `data-validation-element="#validate-code"`.
-- For each kind of validation you'd like to provide feedback on:
-  1. Come up with a name for the validator, e.g. `uppercase`. It shouldn't have any spaces in it.
-  2. On one of the list elements in the `.usa-checklist`, set the `data-validator` attribute to the name of the validator, e.g. `data-validator="uppercase"`. This is the list item that will appear "checked" when the validator's condition is met.
-  3. On the input field, add a field called data-validate-validator name and set its value to a [regular expression](https://regexone.com/) (described on RegexOne) that represents whether the validator's condition is met, e.g. `data-validate-uppercase="[A-Z]"`.
+- **Mirror this validation on the server side.** The validation component is intended primarily for usability, not as a robust security solution, since all the validation logic occurs on the client-side. The validation should be "mirrored" on the server-side for security purposes.
```

Comment:
"client-side" doesn't need to be hyphenated unless it's: "logic occurs client-side."

Reply:
What does "mirrored" mean? Since it's slightly metaphorical, can we clarify exactly what it means in the description?

Reply:
Removed the hyphen and re-wrote some of the copy in https://github.com/uswds/uswds-site/pull/1851/commits/1e3fda7cb66607345fecca510fcaf18df11fd3a6

Let me know if it still needs more clarity.